docs/how-to-build-with-ddn/with-others.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 6
+sidebar_position: 7
```

Comment:
DBs in this section were not sorted alphabetically. For, now just made elastic position 6 and moved this to position 7. Let me know please @robertjdominguez if that's okay

Reply:
Yep, we can alphabetize in a follow-up 👍 

Bumping this as you have is perfect for now.